packages/terra-tag/src/Tag.module.scss

```diff
@@ -39,7 +39,11 @@
     }
 
     &:focus {
-      outline: none;
+      border-color: var(--terra-tags-rollup-tag-focus-border-color, #868a8c);
+      border-width: var(--terra-tags-rollup-tag-focus-border-width, 1px);
+      box-shadow: var(--terra-tags-rollup-tag-focus-box-shadow, none);
+      outline: var(--terra-tags-rollup-tag-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-tags-rollup-tag-focus-outline-offset, 2px);
```

Comment:
Why do we need a border, box shadow, and outline?  It seems like we are using three different approaches in one implementation.

Reply:
Updated Thanks
commit: e43a8de12501cf88bde82bee516607eed4736a70